src/applications/personalization/dashboard/containers/DashboardApp.jsx

```diff
@@ -209,6 +210,8 @@ class DashboardApp extends React.Component {
             {this.renderLOAPrompt()}
             {this.renderMVIWarning()}
 
+            {(document.location.hostname === 'staging.vets.gov') && <AppointmentsWidget/>}
+
```

Comment:
Could possibly move this into a utility file in case this condition starts to getting used a lot